packages/renderer/src/lib/preferences/PreferencesRenderingItem.spec.ts

```diff
@@ -96,3 +96,24 @@ test('props title full should use full record id', async () => {
     expect(element).toHaveClass('font-semibold');
   });
 });
+
+test('locked record should display managed-by label', async () => {
+  const lockedRecord: IConfigurationPropertyRecordedSchema = {
+    id: 'proxy.http',
+    title: 'Proxy',
+    parentId: 'proxy',
+    description: 'HTTP proxy configuration',
+    type: 'string',
+    locked: true,
+  };
+
+  const { getByText } = render(PreferencesRenderingItem, {
+    record: lockedRecord,
+  });
+
+  await vi.waitFor(() => {
+    // Check for "Managed" label text is set
+    const label = getByText('Managed');
+    expect(label).toBeDefined();
```

Comment:
nitpick: no need to check `.toBeDefined()`, because `getByText` will either throw or return a defined label.

```ts
  await vi.waitFor(() =>  getByText('Managed'));
```